tests/test_mlkit.py

```diff
@@ -353,7 +416,131 @@ def test_model_source_validation_errors(self, model_source):
     def test_gcs_tflite_source_validation_errors(self, uri, exc_type):
         with pytest.raises(exc_type) as err:
             mlkit.TFLiteGCSModelSource(gcs_tflite_uri=uri)
-        check_error(err.value, exc_type)
+        check_error(err, exc_type)
+
+
+class TestCreateModel(object):
+    """Tests mlkit.create_model."""
+    @classmethod
+    def setup_class(cls):
+        cred = testutils.MockCredential()
+        firebase_admin.initialize_app(cred, {'projectId': PROJECT_ID})
+        mlkit._MLKitService.OPERATION_POLL_DELAY_SECONDS = 0.1  # shorter for test
+
+    @classmethod
+    def teardown_class(cls):
+        testutils.cleanup_apps()
+
+    @staticmethod
+    def _url(project_id):
+        return BASE_URL + 'projects/{0}/models'.format(project_id)
+
+    @staticmethod
+    def _op_url(project_id, model_id):
+        return BASE_URL + \
+            'operations/project/{0}/model/{1}/operation/123'.format(project_id, model_id)
+
+    def test_create_model_immediate_done(self):
```

Comment:
You can drop _create_model_ part from these test case names. They are already nested under `TestCreateModel`. Use short names like `test_immediate_done`, `test_with_polling`, and `test_operation_error`.

Reply:
done